django/db/models/base.py

```diff
@@ -407,91 +407,102 @@ def __init__(self, *args, **kwargs):
         # overrides it. It should be one or the other; don't duplicate the work
```

Comment:
Do you understand this comment block (from 90acc8ff7e1b27b5c2f7cd5a2440d94d5fa22445)? I'm not sure what code it refers to and if it's correctly positioned (or even still relevant).